content/_includes/birds.macros.njk

```diff
@@ -40,12 +40,14 @@ params:
     ) }}
   {% else %}
     {% for name in names %}
-      {% set tween = ' & ' | mdInline if (loop.length == 2) else ',' %}
+      {{ '&' if loop.last and (loop.length > 2) }}
       {{ author(
         name=name,
         collections=collections,
-        link=link
+        link=link,
+        fullname=false
       ) }}
+      {%- set tween = ' & ' | mdInline if (loop.length == 2) else ',' -%}
```

Comment:
Why does this one go through `mdInline` but the earlier one doesn't?

Reply:
oh, I bet we added that initially to convert `&` -> `<span class="amp">&</span>` with typogr. I'll look into either doing that with both, or I think there might be a simpler way to handle our ampersand typography…

Reply:
What's weird is that it looked to me like both of those ampersands were already being converted anyway 🤷